src/components/BreedsList.vue

```diff
@@ -255,28 +176,16 @@ export default {
     },
   },
   computed: {
-    displayedBreeds() {
-      if (this.$route.name === "search") {
-        return this.searchList;
-      } else {
-        return this.breedsList;
-      }
-    },
+    ...mapState(useStore, [
+      "displayedList",
+      "loadingState",
+      "receivedDataState",
+      "isInputDisabled",
+      "searchQuery",
+    ]),
```

Comment:
Dodal sem ...mapState za pridobivanje state-a oz. gettersov in ...mapActions za manipuliranje state-a... 
A ti kdaj tudi `...mapStores` uporabljaš? Sem ga hotel uporabit, da mi nebi blo treba vseh computed naštevat oz state-ov, pa mi ni še nekak ratalo.
Je nekak to ok?